lib/dao.js

```diff
@@ -2696,14 +2709,19 @@ DataAccessObject.updateAll = function(where, data, options, cb) {
           inst = new Model(data, {applyDefaultValues: false});
         }
 
-        // validation required
-        inst.isValid(function(valid) {
-          if (valid) {
-            doUpdate(ctx.where, ctx.data);
-          } else {
-            cb(new ValidationError(inst));
+        if (doValidate === false) {
```

Comment:
`if (doValidate) {}` and `if (doValidate === false) {` ---> `if (doValidate !== true)`?